web/src/app/history/task-query/task-query.component.html

```diff
@@ -56,7 +56,10 @@ <h4 class="panel-header">Taskana history query</h4>
         <div class="divTableBody">
           <div *ngFor="let task of taskQuery" class="divTableRow">
             <ng-container *ngFor="let taskHeader of taskQueryHeader | mapToIterable">
-              <div class="divTableCell" *ngIf="filterFieldsToShow(taskHeader.key) && !filterExpandGroup(taskHeader.key)">
+              <div class="divTableCell" *ngIf="isDate(taskHeader.key)">
+                {{task[taskHeader.key] | date:'yyyy-MM-dd'}}
```

Comment:
I would use date-time-zone.pipe.ts instead of date pipe. But date-time-zone.pipe.ts is defined in task/831 so you will have to wait until it is merged.

Reply:
Make  sure that the task number matches with jira task number here you have TSK-838 but in jira TSK-842

Reply:
I can change it, but I don't think I need the 'HH:mm:ss'-part.
For me it's easier to understand when I don't see the time.

Reply:
Benni, it's not just showing the correct date format, date-time-zone pipe applies timezone according to the browser location, if you want yo can extend the pipe for adding the date format. but this is up to you.

Reply:
Ok, now I understood.
I try to change it!